src/Refactoring-Core/RBIsVariableNotDefinedInHierarchy.class.st

```diff
@@ -0,0 +1,31 @@
+"
+I check if `name` is not already used by a variable in the `class` hierarchy
+"
+Class {
+	#name : 'RBIsVariableNotDefinedInHierarchy',
```

Comment:
I would suggest to make this condition `RBIsVariableDefinedInTheHierarchy` and then use `not` to get the negated version.

Reply:
I just now realize that will work for `check` but violators will be wrong. I'll see to create not implementation for the violators classes.

Reply:
> I just now realize that will work for `check` but violators will be wrong. I'll see to create not implementation for the violators classes.

Yes, this would be very useful :). For the moment I will keep the `RBIsNotVariableDefinedInTheHierarchy` and I will remove it when we have the violators for `not`